assets/js/blocks/cart-checkout/cart/full-cart/index.js

```diff
@@ -104,38 +104,39 @@ const Cart = () => {
 			} );
 		}
 		if ( ! DISPLAY_PRICES_INCLUDING_TAXES ) {
-			const totalTax = parseInt( cartTotals.total_tax, 10 );
+			const totalTax = parseInt( fetchedCartTotals.total_tax, 10 );
 			totalRowsConfig.push( {
 				label: __( 'Taxes:', 'woo-gutenberg-products-block' ),
 				value: totalTax,
 			} );
 		}
-		const totalShipping = parseInt( cartTotals.total_shipping, 10 );
-		const totalShippingTax = parseInt( cartTotals.total_shipping_tax, 10 );
-		totalRowsConfig.push( {
-			label: __( 'Shipping:', 'woo-gutenberg-products-block' ),
-			value: DISPLAY_PRICES_INCLUDING_TAXES
-				? totalShipping + totalShippingTax
-				: totalShipping,
-			description: (
-				<Fragment>
-					{ __(
-						'Shipping to location',
-						'woo-gutenberg-products-block'
-					) + ' ' }
-					<ShippingCalculator
-						address={ shippingCalculatorAddress }
-						setAddress={ setShippingCalculatorAddress }
-					/>
-				</Fragment>
-			),
-		} );
-
+		if ( isShippingCalculatorEnabled ) {
+			const totalShipping = parseInt( fetchedCartTotals.total_shipping, 10 );
+			const totalShippingTax = parseInt( fetchedCartTotals.total_shipping_tax, 10 );
+			totalRowsConfig.push( {
+				label: __( 'Shipping:', 'woo-gutenberg-products-block' ),
+				value: DISPLAY_PRICES_INCLUDING_TAXES
+					? totalShipping + totalShippingTax
+					: totalShipping,
+				description: (
+					<Fragment>
+						{ __(
+							'Shipping to location',
+							'woo-gutenberg-products-block'
+						) + ' ' }
+						<ShippingCalculator
+							address={ shippingCalculatorAddress }
+							setAddress={ setShippingCalculatorAddress }
+						/>
+					</Fragment>
+				),
+			} );
+		}
 		return totalRowsConfig;
 	};
 
-	const totalsCurrency = getCurrencyFromPriceResponse( cartTotals );
-	const totalRowsConfig = getTotalRowsConfig( cartTotals );
+	const totalsCurrency = getCurrencyFromPriceResponse( fetchedCartTotals );
+	const totalRowsConfig = getTotalRowsConfig( fetchedCartTotals );
```

Comment:
The param is no longer needed (assuming `getTotalRowsConfig` stays inside `Cart` component).